### PR TITLE
Add Solana example for CustomAuth

### PR DESCRIPTION
**Pull request by phuctm97**

### CODE REVIEW

File: files/customauth/web/solana/index.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      <p id="text">Loading...</p>
+      <p id="error"></p>
+      <p id="public-address"></p>
+      <p id="private-key"></p>
+      <button id="login">Login</button>
+    </div>
+    <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/@toruslabs/torus-direct-web-sdk@4.8.0/dist/directWebSdk.umd.min.js"></script>
+    <script src="https://unpkg.com/@solana/web3.js@0.92.0/lib/index.iife.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/@toruslabs/openlogin-utils@0.4.0/dist/openlogin-utils.umd.min.js"></script>
```
Comment (himanshuchawla009) [inline files/customauth/web/solana/index.html:20]: i think key derivation (secpk.. to ed25199) has been moved to different package in openlogin latest version.

https://cdn.jsdelivr.net/npm/@toruslabs/openlogin-ed25519@0.5.0/dist/openlogin-ed25519.umd.min.js

File: files/customauth/web/solana/index.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        $("#text").text("Didn't log in.");
+        $("#login").show();
+      });
+
+      $("#login").click(function () {
+        window.customauth
+          .triggerLogin({
+            typeOfLogin: "google",
+            clientId:
+              "221898609709-obfn3p63741l5333093430j3qeiinaa8.apps.googleusercontent.com",
```
Comment (himanshuchawla009) [inline files/customauth/web/solana/index.html]: set the value of client id as "YOUR_CLIENT_ID"

File: files/customauth/web/solana/index.html (hunk 1 above)
Comment (phuctm97) [reply files/customauth/web/solana/index.html:20]: Yeah, I intentionally used `openlogin-utils` v0.4.0 which still has key derivation. This has nothing to do with OpenLogin, so I think it's good enough?

File: files/customauth/web/solana/index.html (hunk 2 above)
Comment (phuctm97) [reply files/customauth/web/solana/index.html]: Fixed

File: files/customauth/web/solana/index.html (hunk 1 above)
Comment (himanshuchawla009) [reply files/customauth/web/solana/index.html:20]: yeah, as long as u r using 0.4 is it fine, but can break if latest version is used by someone by following this guide.

**Pull request merged**